iot_config/src/region_map.rs

```diff
@@ -46,7 +46,7 @@ impl RegionMap {
     }
 
     pub fn get_region(&self, location: Cell) -> Option<Region> {
-        self.region_hextree.get(location).cloned()
+        self.region_hextree.get(location).unzip().1.copied()
```

Comment:
`get` now returns the actual cell, which is likely a parent cell of `location`. Since we don't seem to need that information, `unzip().1` throws away the actual cell and possible gets optimized away.